src/driver.cpp

```diff
@@ -95,8 +95,9 @@ std::set<std::string> Driver::list_modules() const
         {
           for (auto &match : bpftrace_.probe_matcher_->get_matches_for_ap(*ap))
           {
-            std::string mod = match;
-            auto match_modules = bpftrace_.get_func_modules(erase_prefix(mod));
+            std::string func = match;
+            erase_prefix(func);
+            auto match_modules = bpftrace_.get_func_modules(erase_prefix(func));
```

Comment:
Calling `erase_prefix` twice?

Reply:
Damn, no, that's a mistake. It works b/c it's basically a no-op, but the second `erase_prefix` shouldn't be there. Good catch, fixed.

Reply:
Ah it's because `erase_prefix` returns the prefix and not the modified string - a little prone to errors like this with a signature like that 😄

Reply:
Right, `get_and_erase_prefix` would be a better name. I'll keep that in mind and change it when I work on something related.